sunpy/timeseries/sources/noaa.py

```diff
@@ -56,60 +57,83 @@ class NOAAIndicesTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'noaaindices'
 
-    @peek_show
-    def peek(self, type='sunspot SWO', **kwargs):
+    def plot(self, axes=None, plot_type='sunspot SWO', **kwargs):
```

Comment:
The change to "plot_type" will need a breaking changelog entry.

Reply:
I changed this because type is a keyword, should I revert to type?

Reply:
No, it shouldn't have been `type` in the first place.